src/test/java/io/phasetwo/service/Helpers.java

```diff
@@ -9,11 +10,19 @@
 
 public class Helpers {
 
+  private static final ObjectMapper mapper;
+
+  static {
+    mapper = new ObjectMapper();
+    mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+  }
+
+  public static ObjectMapper objectMapper() {
```

Comment:
Should all ObjectMapper usage rely on this instead of new ObjectMapper in the future ?

Reply:
The problem is that there is a disconnect when making changes to the API request/response objects between what is added here and what has been released in the client library. This stops the tests (which use the client library) from complaining when there is a missing attribute (in this case, the Invitation `attributes`). It's not ideal, but this will stop complaining until we've upgraded the client library with the changes.